cdap-client-tests/src/test/java/co/cask/cdap/client/ArtifactClientTestRun.java

```diff
@@ -82,6 +81,10 @@ public InputStream getInput() throws IOException {
   public void setUp() throws Throwable {
     super.setUp();
     artifactClient = new ArtifactClient(clientConfig, new RESTClient(clientConfig));
+    for (ArtifactSummary artifactSummary : artifactClient.list(Id.Namespace.DEFAULT, false)) {
```

Comment:
this is more of a catch all to clean up state since there is no delete all in a namespace.  A lot of other tests in the suite don't know about artifacts, they just deploy apps, which leaves those artifacts around even if the app is deleted.